lib/Mojolicious/Command/eval.pm

```diff
@@ -48,7 +55,9 @@ Mojolicious::Command::eval - Eval command
 
 =head1 DESCRIPTION
 
-L<Mojolicious::Command::eval> runs code against applications.
+L<Mojolicious::Command::eval> runs code against applications. If the result is
+a L<Mojo::Promise>, it will wait until the promise is completed and the result
```

Comment:
The documentation for `Mojo::Promise::wait` says "fulfilled or rejected" instead of "completed".

Reply:
I can make this change the next time I'm at a computer